src/Analyzers/shared/FeatureDetection/Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
```suggestion
    <!-- This does not represent the TFM for the code. It's only here because /t:Pack requires it. -->
    <TargetFramework>netstandard1.0</TargetFramework>
    <IsPackable>true</IsPackable>
    <IsShipping>false</IsShipping>
    <NoBuild>true</NoBuild>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
    <EnableDefaultItems>false</EnableDefaultItems>
    <IncludeBuildOutput>false</IncludeBuildOutput>
    <IncludeSymbols>false</IncludeSymbols>
    <ContentTargetFolders>contentFiles</ContentTargetFolders>
    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
    <DefaultExcludeItems>$(DefaultExcludeItems);$(BaseOutputPath);$(BaseIntermediateOutputPath);</DefaultExcludeItems>
    <NoWarn>$(NoWarn);NU5105;CS8021</NoWarn>
    <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="$(MSBuildProjectDirectory)\**\*.cs" Exclude="$(DefaultExcludeItems)">
      <Pack>true</Pack>
      <PackagePath>$(ContentTargetFolders)\cs\netstandard1.0\shared\</PackagePath>
    </Compile>
  </ItemGroup>
```

Here is the goop to make this a .Sources packages, contain only content, not a binary.

Reply:
My goal was to have this project still compile in VS. It looks like if this is going to be ns10 then that is busto. Should I remove the package references and remove from the solution?

Reply:
We haven't tried to do that with other .Sources packages because there wasn't any demand. Typically code in a .Sources package are compiled by at least one other project in the repo (and if they code isn't...then I have to ask, why is it here?)

But if you want to try and make it work, that's okay with me.